tests/FixerFactoryTest.php

```diff
@@ -387,4 +387,22 @@ private function createFixerMock($name, $priority = 0)
 
         return $fixer;
     }
+
+    /**
+     * @dataProvider provideConflictingFixersRules
+     * @expectedException \UnexpectedValueException
+     * @expectedExceptionMessageRegExp #^The rules contain conflicting fixers:\n#
+     */
+    public function testConflictingFixers(RuleSet $ruleSet)
+    {
+        FixerFactory::create()->registerBuiltInFixers()->useRuleSet($ruleSet);
+    }
+
+    public function provideConflictingFixersRules()
+    {
+        return array(
+            array(new RuleSet(array('short_array_syntax' => true, 'long_array_syntax' => true))),
+            array(new RuleSet(array('long_array_syntax' => true, 'short_array_syntax' => true))),
```

Comment:
Actually, I would like to merge this PR soon, if you want to work on it it would be nice, but please do it in separate PR ;)